modules/gdnative/nativescript/nativescript.cpp

```diff
@@ -1738,8 +1739,6 @@ void NativeReloadNode::_notification(int p_what) {
 			}
 
```

Comment:
While here, you can remove that empty line before the final `} break;` (same of `_FOCUS_OUT`),